File: Java/Functions/functionOverloading.java
public class functionOverloading {

    
    public static int sum(int numOne,int numTwo) {

        return numOne+numTwo;
        
    }
    public static int sum(int numOne,int numTwo,int numThree) {

        return numOne+numTwo+numThree;
        
    }
    public static void main(String[] args) {
        System.out.println("sumThree ="+ (sum(10, 10, 10)));
        System.out.println("sumTwo="+ sum(10, 10));
        
    }
}

File: Cpp/Functions/functionOverloading.cpp
#include <iostream>

using namespace std;

int sum(int numOne, int numTwo) {
  return numOne + numTwo;
}

int sum(int numOne, int numTwo, int numThree) {
  return numOne + numTwo + numThree;
}

int main() {
  cout << "sumThree = " << sum(10, 10, 10) << endl;
  cout << "sumTwo = " << sum(10, 10) << endl;

  return 0;
}
